src/poliastro/czml/extract_czml.py

```diff
@@ -138,34 +82,28 @@ def _init_orbit_packet_cords_(self, i):
             cords = position.represent_as(CartesianRepresentation).xyz.to(u.meter).value
             cords = np.insert(cords, 0, h.value * k, axis=0)
 
-            self.czml[i]["position"]["cartesian"] += list(
-                map(lambda x: x[0], cords.tolist())
-            )
+            cart_cords += list(map(lambda x: x[0], cords.tolist()))
```

Comment:
Is there a more straightforward way to write this? It's a bit cryptic :sweat:

Reply:
This simply flattens the list (since ``cords.tolist()`` returns a list of lists). I guess we could also use something like ``itertools.chain``?

Reply:
Oh, I see! I think `itertools.chain` is better recognized as "flattening the list" but, whether you change it or not, please add a comment explaining